client/src/test.ts

```diff
@@ -9,8 +9,8 @@ import {
 
 declare const require: {
   context(path: string, deep?: boolean, filter?: RegExp): {
-    keys(): string[];
     <T>(id: string): T;
+    keys(): string[];
```

Comment:
The new version of ESLint asked us to reorder these declarations.

Reply:
Interesting. Any idea why?

Reply:
Right, yes—the declaration at the top overloads the function call operator `()`

(It's the TypeScript equivalent of C++'s `operator()(string id)`.)

Since it's such a special declaration, ESLint wants it to have top billing.